src/diffusers/pipelines/pag/pag_utils.py

```diff
@@ -84,7 +84,7 @@ def get_block_type(module_name):
         def get_block_index(module_name):
```

Comment:
Could you revert the changes here so I don't have a merge conflict to resolve in #8936? :sweat_smile: It's taken care of in the PAG refactor

Reply:
@a-r-r-o-w Hey, sure! Just reverted the changes